app/scripts/comp/launcher-electron.js

```diff
@@ -244,6 +244,9 @@ const Launcher = {
             }
         }
         return ps;
+    },
+    openFileChooser: function(callback, button) {
```

Comment:
If the whole feature is represented as a single method, I think it's pretty ok to check it directly. :+1: